recipes/polyhedral-gravity-model/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "polyhedral-gravity-model" %}
+{% set org = "esa" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "2255b2cd136cfd0e7f02bb1020f67af4ca2c99e23e4006e18077e10dc0a326ac" %}
```

Comment:
`org` and `sha256` are only used once. I don't think using a variable for those improve the recipe.

Reply:
If one needs to increase the version number, one just needs to edit the first lines. This makes the whole thing more maintainable and less error-prone in forgetting the `sha256`. I would prefer to leave it as it is if this is ok.

Reply:
conda-forge has a bot that automatically creates a PR when a new version is detected. It takes care of updating the version variable and the sha in the source section. I'm not sure it handles properly the sha256 if defined as a variable (it might). I'd recommend removing it.